packages/calcite-components/src/components/navigation-logo/navigation-logo.tsx

```diff
@@ -107,34 +113,43 @@ export class CalciteNavigationLogo implements LoadableComponent {
     return <calcite-icon class={CSS.icon} flipRtl={this.iconFlipRtl} icon={this.icon} scale="l" />;
   }
 
+  renderHeaderContent(): VNode {
+    const { heading, headingLevel, description } = this;
+    const headingNode = heading ? (
+      <Heading
+        class={{
+          [CSS.heading]: true,
+          [CSS.standalone]: !this.description,
+        }}
+        key={CSS.heading}
+        level={headingLevel}
+      >
+        {heading}
+      </Heading>
+    ) : null;
+
+    const descriptionNode = description ? (
+      <span class={CSS.description} key={CSS.description}>
+        {description}
+      </span>
+    ) : null;
+
+    return headingNode || descriptionNode ? (
+      <div class={CSS.container}>
```

Comment:
nitpick: should we give this one a key for consistency? `<div class={CSS.container} key={CSS.container}>`. up to you :)